src/client/app/components/about.ts

```diff
@@ -15,15 +15,15 @@ let { config } = PLAYGROUND;
                     <div class="profile__tertiary-text ms-font-m">User ID: ${storage.user}</div>
                     <div class="about__secondary-text ms-font-l">Version: {{config?.build?.version}}
                         <br/><span class="ms-font-m">(Deployed {{config?.build?.humanReadableTimestamp}})</span>
-                        <br/><span class="ms-font-m">{{config?.editorUrl}}</span>
                     </div>
-                    <div class="about__dropdown">
-                        <label class="ms-font-m about__dropdown-label">${Strings.aboutChangeEnvironment}</label>
-                        <select class="about__dropdown-select ms-font-m" [(ngModel)]="selectedConfig" (change)="changeConfig($event.target.value)">
+                    <pre class="about__tertiary-text ms-font-m">{{cache}}</pre>
```

Comment:
Not sure what it looks like without a screenshot showing it, but I'll trust your artistic judgement...